website/src/components/webinars/WebinarLayout.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react'
+
+import classNames from 'classnames'
+import styles from './WebinarLayout.module.scss'
+
+import { ContentSection } from '../content/ContentSection'
+import { breakpoints } from '../../breakpoints'
+import { useWindowWidth } from '../../hooks'
+import { useHubSpot } from '../../hooks/hubSpot'
+import { useChiliPiper } from '../../hooks/chiliPiper'
+
+interface Props {
+    customer?: Customer
+    title: string
+    subtitle: string
+    description: React.ReactNode
+    speakers: Speaker[]
+    children?: React.ReactNode
+}
+
+interface Customer {
+    name: string
+    logo: string
+    href: string
+}
+
+interface Speaker {
+    name: string
+    title: string
+    img: string
+    bio: string
+}
+
+export const WebinarLayout: React.FunctionComponent<Props> = ({
+    title,
+    subtitle,
+    customer,
+    description,
+    speakers,
+    children,
+}) => {
+    useHubSpot({
+        portalId: '2762526',
+        formId: '66361163-5e08-4be3-8ab0-6590b70df69e',
```

Comment:
I notice that the `formId` appears to be different in the issue description: https://github.com/sourcegraph/about/issues/5319

>portalId: "2762526", 
>formId: "abb86998-1a6a-4cfd-9888-ffba09c850c9"

It looks like `66361163-5e08-4be3-8ab0-6590b70df69e` is the `formId` listed in the upcoming Code Insights webinar issue https://github.com/sourcegraph/about/issues/5317.

Because there are different `formId` values for each instance of this template, perhaps we can pass in `formId` as a prop? It looks like `portalId` is always `2762526`.

Reply:
Good catch, Tim! Updated this `formId` prop in [d2b6b38](https://github.com/sourcegraph/about/pull/5344/commits/d2b6b3864831d5f7fddde9804aa8bb66feb5623e)

Reply:
Awesome, thanks Becca! This looks good.